src/main/zapHomeFiles/hud/posthtml.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  plugins: {
+    "posthtml-expressions": {
+      locals: {
+        ZAP_HUD_FILES: process.env.ZAP_HUD_FILES,
```

Comment:
whats the reason for the env var pattern here instead of just hard coding them?

Reply:
Parcel supports `.env` files by default and it will do variable substitution and logic evaluation throughout the app at `build`.  Then PostHTML has it's own requirements for it's own `posthtml--expressions` plugin, which this config file is for handling it's own HTML interpolation which i wouldn't be able to do in a template with `process.env` w/o PostHTML (cuz only can be in JavaScript).

https://parceljs.org/env.html
https://github.com/posthtml/posthtml-expressions

Reply:
@jaywon: fwiw each of those instances of `it's own` in https://github.com/zaproxy/zap-hud/pull/459#discussion_r269415880 should be `its own`.

(I know it isn't particularly material here, but at some point I'm probably going to be running a grammar check against some of these repositories, so to the extent I can remind people to be aware of the distinction: `it's` => `it is` / `its` => possessive, there's a return on my investment.)